.github/workflows/website.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     - uses: EndBug/add-and-commit@v9
       if: env.GIT_DIRTY != null && env.GIT_DIRTY != '0'
       with:
-        add: 'content/news'
+        add: '-A content/news'
```

Comment:
Is this change a debug leftover?

Reply:
~~No its required because in the case where the script does the line ending fixing, it essentially changes the entire file. With the renaming in the same step, git will think that the `XXXX-XX-XX-*` file has been deleted and a new one created. The `-A` ensures that it stages the "new" file as well as the "deleted" one. At that point it also realizes again that that was in fact just a renaming:~~

I just tested it again (this time passing `content/news` instead of `content/news/*`) and it picked it up. So this is in fact unnecessary. I will remove.